recipes-samples/images/rpb-console-image.bb

```diff
@@ -1,21 +1,12 @@
-SUMMARY = "Basic console image"
-
-IMAGE_FEATURES += "splash package-management debug-tweaks ssh-server-openssh hwcodecs tools-debug"
-
-LICENSE = "MIT"
+require rpb-minimal-image.bb
 
-inherit core-image distro_features_check extrausers
+SUMMARY = "Basic console image"
 
-# let's make sure we have a good image..
-REQUIRED_DISTRO_FEATURES = "pam systemd"
```

Comment:
i don't see this setting done anywhere after your change

Reply:
Already set at rpb-minimail-image.bb.